### PR TITLE
Create indents.scm for python

### PR DESCRIPTION
**Pull request by esn89**
Create indents.scm for python

### CODE REVIEW

File: queries/python/indents.scm [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  (try_statement)
+  (for_statement)
+  (for_in_clause)
+  (class_definition)
+] @indent
+
+[
+  "elif"
+  "else"
+] @branch
```
Comment (kyazdani42) [inline queries/python/indents.scm:14]: could there be things like finally clause here ? or catch ?

Comment (esn89) [reply queries/python/indents.scm:16]: I have added finally and "except", since python doesn't have catch

**Review by theHamsta — APPROVED** [review]
Looks good to me. I can not really judge this since the feature is not working on my machine.

**Comment by esn89** [comment]
@theHamsta @kyazdani42 

Thanks.  I just updated the plugin via vim-plug and Notice:

```
Error detected while processing /Users/esn89/.config/nvim/plugged/nvim-treesitter/plugin/nvim-treesitter.vim:                                                                                                                             
line   12:                                                                                                                                                                                                                               
E5108: Error executing lua ...nvim-treesitter/lua/nvim-treesitter/query_predicates.lua:89: attempt to call field 'add_directive' (a nil value)                                                                                           
Press ENTER or type command to continue                                                                                        
```

Was wondering, if this is due to my latest change or somethign else

**Comment by doubleloop** [comment]
Anyone tested this? Should it work? Just recompiled master vim, updated plugin and parsers then started some python programming and immediately had to disable treesitter indent in config because it is difficult to write code with this (nearly every indentation is either too big or there is no indentation at all, or typing `:` makes properly indented line more indented..)
Is it known issue or did I screw up configuration? All this indenting issues occur with minimal configuration:

```
call plug#begin('~/.local/share/nvim/plugged')
Plug 'nvim-treesitter/nvim-treesitter'
call plug#end()

lua << EOF
require('nvim-treesitter.configs').setup {
    highlight = {
        enable = true
    },
    indent = {
        enable = true,
    },
}
EOF
```

**Comment by theHamsta** [comment]
Indentation support is only alpha right now.

**Pull request merged**